logstash-core/lib/logstash/pipeline.rb

```diff
@@ -415,10 +430,13 @@ def flush_filters(options = {}, &block)
   end
 
   def start_flusher
+    # Invariant to help detect improper initialization
+    raise "Attempted to start flusher on a stopped pipeline!" if stopped?
```

Comment:
No, there isn't, but I felt this behavior was important enough that a future-proofing invariant was called for. This may catch a dumb mistake down the road.